.github/workflows/node-ci.yml

```diff
@@ -0,0 +1,41 @@
+name: Node.js CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [14.x]
```

Comment:
This can be removed, and the node version can be inlined in the `with` > `node-version` below, e.g. [see here](https://github.com/opentripplanner/otp-react-redux/blob/dev/.github/workflows/node-ci.yml#L19)

Reply:
Good catch thank you! Fixed in 1711f39